docs/networks/mainnet/running-a-node.md

```diff
@@ -89,14 +86,123 @@ After the initialization is successful, we must download the blockchain clients,
 $ sudo lukso install
 ```
 
+#### Setting your Public IP Address
+
+Your public IP address is a unique identifier assigned to your internet connection by your service provider. Every device connected to the public internet is set as an IP address for communication with other devices.
+
+To find other nodes in the network, your public IP should be exposed to build solid peer connections and download data more quickly. This Public IP value must be set within the consensus client configuration file that the CLI downloads from the [LUKSO Network Configuration](https://github.com/lukso-network/network-configs) repository.
+
+<Tabs>
+  <TabItem value="prysm-config" label="Prysm">
+
+Within the node's working directory, open up the related configuration file with an editor of your choice:
+
+```text
+/configs/mainnet/prysm/prysm.yaml
```

Comment:
this file may not be here, maybe adding smtgh like:

```
[pathToCliConfigFolder]/mainnet/prysm/prysm.yaml
```

would make it more obvious this can change depending on configs